src/test/java/com/easypost/Fixture.java

```diff
@@ -5,13 +5,9 @@
 import java.util.List;
 import java.util.Map;
 
-final class Fixture {
+public abstract class Fixture {
```

Comment:
Why did this get switched to `abstract`? Is it being extended/re-implemented somewhere else, and if so, why?

Reply:
This class is being used in the unit tests. Changed it to `abstract` to prevent instantiation.

Reply:
Fair enough